apps/team-awesome-web/src/components/header/Header.tsx

```diff
@@ -53,7 +52,7 @@ export const Header = () => {
       <Styled.RightMenu>
         <Styled.Profile>
           <HumanoidsLogo />
-          <Image src="/img/profile.jpg" alt="Avatar image" width={36} height={36} />
+          <Styled.Avatar src="img/avatars/amijs.png" alt="Avatar image" width={36} height={36} />
```

Comment:
You can remove the width and height styles here as these are already declared in your styled component.

Reply:
Whoops, those were indeed mandatory for `<Image>` but I forgot to drop them with the change to a style component 😓